test/libsolidity/syntaxTests/iceRegressionTests/oversized_var.sol

```diff
@@ -4,11 +4,16 @@ contract b {
     }
 
     c d;
+
     function e() public view {
         c storage x = d;
-	x.a[0];
+        x.a[0];
+        function()[3**44] storage b;
     }
 }
 // ----
+// Warning 2519: (152-179): This declaration shadows an existing declaration.
 // Warning 3408: (66-69): Variable "d" covers a large part of storage and thus makes collisions likely. Either use mappings or dynamic arrays and allow their size to be increased only in small quantities per transaction.
-// Warning 2332: (110-111): Type "b.c" covers a large part of storage and thus makes collisions likely. Either use mappings or dynamic arrays and allow their size to be increased only in small quantities per transaction.
+// Warning 2332: (111-112): Type "struct b.c" covers a large part of storage and thus makes collisions likely. Either use mappings or dynamic arrays and allow their size to be increased only in small quantities per transaction.
+// Warning 2332: (152-169): Type "function ()[984770902183611232881]" covers a large part of storage and thus makes collisions likely. Either use mappings or dynamic arrays and allow their size to be increased only in small quantities per transaction.
```

Comment:
This error is weird though. How can you declare a static storage array inside a function?

Reply:
It should be a storage pointer.

Reply:
I wonder why those trigger this error but these don't:
```
contract C {
    uint[2**255] b;
    function f() public {
        uint[2**255] storage a;
    }
}
```

Reply:
Ah my bad, they do, I just thought they were errors but they are warnings.

Reply:
Now I'm wondering why these don't complain about uninitialized storage pointer

Reply:
I think the error happens when you try to use an uninitialized storage pointer. This case is fine I think.